### DISCUSSION TITLE
How is cloning currently implemented, like https://sortablejs.github.io/Vue.Draggable/#/clone

### DISCUSSION THREAD

**Discussion by litingyes** [atlassian/pragmatic-drag-and-drop#34] [Q&A] [answered]
How is cloning currently implemented, like https://sortablejs.github.io/Vue.Draggable/#/clone

**Comment by alexreardon** [comment, marked as answer]
- By default the native drag preview is a clone of the dragging element (or you can use a custom drag preview)
- You can use `dropEffect: "copy"` on your drop target to make it super clear that your item is being copied (and not moved)
- After a drop, you control what the drop does. So after a drop, if you want to copy / clone, you can make a copy in the new destination and leave the original in place

Let me know how you go!
